specs/BUCK

```diff
@@ -745,7 +745,11 @@ osquery_gentable_cxx_library(
             "windows",
         ),
         (
-            "windows/pipes.table",
+            _SPECS_LOCATION + "/windows/hvci_status.table",
```

Comment:
Remove the `_SPECS_LOCATION` bit here, make it look like the ones above, I think CI is failing over this.